package.json

```diff
@@ -10,15 +10,23 @@
     "exifreader": "^3.11.2",
     "glob": "^7.1.6",
     "lodash": "^4.17.15",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
```

Comment:
Why you have moved React to dependencies? Did you run into some issue?

It's not needed in dependencies as it is only used to build the the `bundle.js` and then that is used by Electron. So only dependencies required by Electron/Node needs to be in dependencies.

It's important to keep this sorting to devDependencies and dependencies in good fit because it affects to the size of our app package which is downloaded and installed by our users.

Reply:
Ah, so dependences should include only electron/node dependences. All other in devDeps? Yeah, I have to remember that this is not a fe project. I'll fix.